packages/dds/tree/src/test/feature-libraries/modular-schema/modularChangeFamily.spec.ts

```diff
@@ -99,6 +101,47 @@ const singleNodeField = new FieldKindWithEditor(
 	new Set(),
 );
 
+const removedRootsFieldKind: FieldKindIdentifier = brand("HasRemovedRootsRefs");
```

Comment:
Once you update the API for the two functions, this stuff can all go back to the way it was, and you don't have to use any of it.